modules/mono/editor/Godot.NET.Sdk/Godot.SourceGenerators/ScriptMethodsGenerator.cs

```diff
@@ -147,6 +151,12 @@ INamedTypeSymbol symbol
 
             foreach (string methodName in distinctMethodNames)
             {
+                source.Append("        /// <summary>\n")
+                    .Append("        /// Cached name for the '")
+                    .Append(methodName)
+                    .Append("' method.\n")
```

Comment:
```suggestion
                    .Append($"        /// Cached StringName for the '{methodName}' method.\n")
```

or even

```suggestion
                    .Append($"        /// Cached <see cref=\"Godot.StringName\" /> for the <see cref=\"{methodName}\" /> method.\n")
```

(same for the other such cases)

Reply:
I think the better readability is worth the performance impact, but it doesn't really matter too much either way so keeping it consistent with the remaining code is probably better.